src/Lykke.Service.ClientSearch/appsettings.json

```diff
@@ -0,0 +1,4 @@
+﻿
+{
+  "SettingsUrl": ""
```

Comment:
Fill the file according the AppSettings with a structure according https://github.com/LykkeCity/Lykke.Service.ClientSearch/blob/9e0cf65d926b9cebc5b12eaaa0f5a20b45047ee8/src/Lykke.Service.ClientSearch.Core/AppSettings.cs

Reply:
structure added